business-networks-workflows/src/main/kotlin/net/corda/bn/flows/DeleteMembershipAttributeChangeRequestFlow.kt

```diff
@@ -45,17 +45,20 @@ class DeleteMembershipAttributeChangeRequestFlow(
             authorise(networkId, bnService) { it.canModifyRoles() }
         }
 
-        val participants = membershipChangeRequestData.participants
+        // build transaction
+        val signers = membershipChangeRequestData.participants.mapNotNull {
```

Comment:
What's the difference between Participants and Signers here? Looks like Signers= participants - ourIdentity?

Reply:
participants was a List<AbstractParty>, while signers is a List<Party>
as for the second question, I forgot to remove ourIdentity.. will do that now, but yes signers should be participants - ourIdentity